pgmpy/models/LinearGaussianBayesianNetwork.py

```diff
@@ -432,3 +433,110 @@ def is_imap(self, JPD):
         raise NotImplementedError(
             "is_imap method has not been implemented for LinearGaussianBayesianNetwork."
         )
+
+    @staticmethod
+    def get_random(
+        n_nodes=5,
+        edge_prob=0.5,
+        node_names=None,
+        n_states=None,
```

Comment:
@kollisaisiddartha `n_states` is not required for LinearGaussianBayesianNetwork. It is required by TabularCPDs which is only for BayesianNetwork.